app/Presenters/UploadedFilesPresenter.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace App\Presenters;
+
+/**
+ * Class AccessoryPresenter
+ */
+class UploadsPresenter extends Presenter
+{
+    /**
+     * Json Column Layout for bootstrap table
+     * @return string
+     */
+    public static function dataTableLayout($object)
+    {
+
+        if ($object =='assets') {
```

Comment:
I’m not sure this is used under the new system - might be able to get rid of the `$object` parameter alltogether?

Reply:
It's vestigial for the changes to come. I can pull it for now tho.